platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/SecureCxfClientFactory.java

```diff
@@ -101,6 +101,15 @@
 
   private static final Integer DEFAULT_RECEIVE_TIMEOUT = 60000;
 
+  private static final String AUTO_REDIRECT_ALLOW_REL_URI = "http.redirect.relative.uri";
+
+  private static final String AUTO_REDIRECT_MAX_SAME_URI_COUNT = "http.redirect.max.same.uri.count";
+
+  // a seemingly magic number.  Needs at least 2 so that it can come back with authentication
+  // cookies.  Set one higher for safety.  The trade-off is failing to catch a loop vs cutting
+  // off a server before it is fully authenticated.  Any small number seems to be a safe option
+  private static final int SAME_URI_REDIRECT_MAX = 3;
```

Comment:
Should this be configurable, just in case 3 is too small for some usecases?

Reply:
We can do that